topics/assembly/tutorials/assembly-with-preprocessing/tutorial.md

```diff
@@ -0,0 +1,910 @@
+
+- a final assembly in FASTA format
+- an assembly graph
+
+Of these, the assembly graph is more information-rich because it not only
+contains the sequences of *all* assembled fragments (including the ones shorter
+than the threshold length defined for inclusion of the fragments into the FASTA
+output), but also indicates the relative average coverage of the fragments by
+sequenced reads and how some of the fragments could potentially be bridged
+after resolving ambiguities manually.
+
+### Assembly inspection with Bandage
+
+On the downside, the assembly graph format takes some getting used to before
+you can make sense out of the information it provides.
+
+This issue can be alleviated through the use of **Bandage**, a package for
+exploring assembly graphs through summary reports and visualizations of their
+contents.
+
+> ### {% icon hands_on %} Hands-on: Assembly stats and visualization with Bandage
+>
+> 1. **Bandage Info** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Graphical Fragment Assembly"*: the assembly graph dataset produced by
+>      **Unicycler**
+>    - *"Output the information in a single tab-delimited line starting with the graph file"*:
+>      `No`
+>
+> 2. **Bandage Image** {% icon tool %} with the following parameters
+>    - {% icon param-file %} *"Graphical Fragment Assembly"*: the assembly graph dataset produced by
+>      **Unicycler**
+>    - *"Node name labels?"*: `Yes`
+>    - *"Node length labels?"*: `Yes`
+>
+{: .hands_on}
+
+Let us inspect the summary report produced by **Bandage Info** {% icon tool %}
+first:
+
+You may be rather disappointed by the large *percentage of of dead ends* in the
+assembly graph (in general, lower is better here), and by the correspondingly
+large *node count*. After all, should the viral sequence not be encoded on a
+single small contig (a quick check at
+[Wikipedia](https://en.wikipedia.org/wiki/Coronavirus#Genome) reveals that
+coronaviruses have genomes in the size range of 30kb)?
+
+On the other hand, there is the *Longest node* of 29768 bp of assembled
+sequence, which is suspiciously close to the expected genome size, but a much
+larger *Estimated sequence length*.
+
+Next, take a look at the assembly graph visualization generated by **Bandage
+Image** {% icon tool %} to see if that tells us more:
+
+Indeed, this output shows that **Unicycler** {% icon tool %} managed to
+assemble a good number of contigs of moderate size, then had trouble with a
+number of really small fragments that it could only assemble with lots of
+ambiguities (leading to that ugly clutter of nodes in the top row of the
+image). Those small fragments will probably be hard to make sense of, but the
+manageable list of moderate-size contigs (nodes 1-23, 25, 26) is encouraging.
+One of them, node 2, is also outstanding since **Unicycler** claims it is
+circular (not that we expect this for the CoV-2 genome though).
```

Comment:
maybe you should take the negation out of the bracket and make the sentence easier to understand?